scim-client/src/test/java/org/apache/directory/scim/client/filter/FilterBuilderGreaterTest.java

```diff
@@ -62,99 +62,99 @@
   public void testGreaterThanT_Int(Integer arg) throws UnsupportedEncodingException, FilterParseException {
     
     String encoded = FilterClient.builder().greaterThan("dog.weight", arg).toString();
-    Filter filter = new Filter(decode(encoded));
+    new Filter(decode(encoded));
```

Comment:
These were all "Unused Local" warnings.  As they are in test classes and exceptions will still get thrown, I have removed the assignment.